app-config.yaml

```diff
@@ -43,6 +43,16 @@ integrations:
     #   apiBaseUrl: https://ghe.example.net/api/v3
     #   token: ${GHE_TOKEN}
 
```

Comment:
You are going to need to add

```
dangerouslyAllowUnverifiedAuth: true
```

To the top level app-config.yaml. This is because backstage disabled the ability to interact with backstage if you aren't logged in as a user with a catalog entity. Adding this in the dev repo allows that.